server/src/lib/api.ts

```diff
@@ -91,7 +95,12 @@ export default class API {
 
     router.get('/language/accents/:locale?', this.getAccents);
     router.get('/language/variants/:locale?', this.getVariants);
-    router.post('/language/request', this.sendLanguageRequest);
+    router.post(
+      '/language/request',
+      googleReCAPTCHAMiddleware,
```

Comment:
nit: function name doesn't need to have "middleware" since it is a middleware (kinda like calling `const dataVariable = <blah>`)

Reply:
Good spot thank you